.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Feature Request
+about: Suggest a new functionality or data change.
+title: ""
+labels: "user request"
+assignees: ""
+---
+
+## Description
+
+Provide a detailed description of the request.
```

Comment:
Since this section is asking for _detailed_ description, we might not need the "ideal behavior" section below. Alternately, the "ideal behavior" section could be changed to something like "What other alternatives have you considered?"